docs/learn/technology/what-is-swarm.md

```diff
@@ -11,59 +11,56 @@ Swarm is peer-to-peer network of nodes which work together to provide decentrali
 
 Swarm can be divided into four main parts:
 
-1. Underlay Network - A peer-to-peer network protocol to serve as underlay transport. 
+1. Underlay Network - A peer-to-peer network protocol to serve as underlay transport.
 2. Overlay Network - An overlay network with protocols powering a distributed immutable storage of chunks (fixed size data blocks).
 3. Data Access Layer - A component providing high-level data access and defining APIs for base-layer features.
 4. Application Layer - An application layer defining standards and outlining best practices for more elaborate use cases.
 
-
 ### 1. Underlay Network
 
-The first part of Swarm is a peer-to-peer network protocol that serves as the underlay transport. The underlay transport layer is responsible for establishing connections between nodes in the network and routing data between them. It provides a low-level communication channel that enables nodes to communicate with each other directly, without relying on any centralised infrastructure.
-
-Swarm is designed to be agnostic of the particular underlay transport used, as long as it satisfies certain requirements.  
-
-1. Addressing – Nodes are identified by their underlay address.
-2. Dialling – Nodes can initiate a direct connection to a peer by dialing them on
-their underlay address.
-3. Listening – Nodes can listen to other peers dialing them and can accept incoming
-connections. Nodes that do not accept incoming connections are called light
-nodes.
-4. Live connection – A node connection establishes a channel of communication which
-is kept alive until explicit disconnection, so that the existence of a connection
-means the remote peer is online and accepting messages.
-5. Channel security – The channel provides identity verification and implements
-encrypted and authenticated transport resisting man in the middle attacks.
-6. Protocol multiplexing – The underlay network service can accommodate several
-protocols running on the same connection. 
-7. Delivery guarantees – Protocol messages have guaranteed delivery, i.e. delivery
-failures due to network problems result in direct error response. Order of delivery
-of messages within each protocol is guaranteed. 
-8. Serialisation – The protocol message construction supports arbitrary data structure
-serialisation conventions.
+Swarm serves as an underlay transport by establishing connections between nodes in the network and routing data between them. This provides a low-level communication channel that enables nodes to communicate with each other directly, without relying on any centralised infrastructure.
```

Comment:
Swarm includes both underlay and overlay, so it is not really accurate to say that Swarm itself serves as an underlay transport. It includes both these layers.